src/gluonts/evaluation/metrics.py

```diff
@@ -86,44 +77,31 @@ def mase(
 
     https://www.m4.unic.ac.cy/wp-content/uploads/2018/03/M4-Competitors-Guide.pdf
     """
-    if exclude_zero_denominator and np.isclose(seasonal_error, 0.0):
-        return np.nan
-
-    return nan_if_masked(np.mean(np.abs(target - forecast)) / seasonal_error)
+    return np.mean(np.abs(target - forecast)) / seasonal_error
 
 
 def mape(
-    target: np.ndarray, forecast: np.ndarray, exclude_zero_denominator=True
+    target: np.ndarray, forecast: np.ndarray, exclude_zero_denominator=False
```

Comment:
`exclude_zero_denominator` could actually be removed?

Reply:
Thanks, it should go indeed.